pages/linux/scrot.md

```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Screen capture utility.
+
+- Capture a screenshot:
+
+`scrot`
+
+- Capture a screenshot interactively:
+
+`scrot --select`
+
+- Capture a screenshot from the currently focued window:
```

Comment:
I believe it should be "focused" insted of focued.

Reply:
Thank you, fixed it.